Tests/TuistSupportTests/Extensions/String+ExtrasTests.swift

```diff
@@ -17,6 +17,39 @@ final class StringExtrasTests: TuistUnitTestCase {
         // Then
         XCTAssertEqual(got, "frameworkIOSResources")
     }
+
+    func test_string_doesnt_match_URL_regex() {
+        // Given
+        let stringToEvaluate = "not a url string"
+
+        // When
+        let result = stringToEvaluate.isGitURL
+
+        // Then
+        XCTAssertFalse(result)
+    }
+
+    func test_string_does_match_http_URL_regex() {
+        // Given
+        let stringToEvaluate = "https://github.com/tuist/ExampleTuistTemplate.git@develop"
```

Comment:
I'd add tests for the following URLs:
- `https://github.com/tuist/ExampleTuistTemplate`
- `https://github.com/tuist/ExampleTuistTemplate@develop`